airmozilla/base/templates/email_base.html

```diff
@@ -1,47 +1,78 @@
 <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
 <html>
-
+ 
```

Comment:
What this means is that your editor is misconfigured. Or something. Here is a line that simply only contains a space and a newline character. I.e. ` \n`. 

Most editors have it so that when you save, it automatically trim all trailing whitespace on all lines. Which editor are you using?